README.md

```diff
@@ -9,22 +11,39 @@ Issues and feature requests posted to GitHub are not stored in any
 publicly available location, so I find that I quickly lose track of
 the things that I've sent them.
 
+## For GitHubbers
+
 If you are a GitHubber, and you would like to respond and close issues
 on this repository, please let me (or one of the other collaborators)
 know, and we'll add you as a collaborator.  Or maybe use some secret
 GitHub internal APIs to get access, if you have such things.  If you
 want to know what people are asking for, or ask for them to provide
 details about use cases etc, then head over to the issues list.
 
-Post an issue if you have an issue or feature request for GitHub.
+
+## If you have an issue or feature request for GitHub.
+
+1. Search for existing [issues](https://github.com/isaacs/github/issues) here first
```

Comment:
You can drop the `first` at the end, it's already the first item in an ordered list.